osgp/protocol-adapter-dlms/osgp-protocol-adapter-dlms/src/main/java/org/opensmartgridplatform/adapter/protocol/dlms/application/services/ThrottlingService.java

```diff
@@ -0,0 +1,113 @@
+/**
+ * Copyright 2020 Smart Society Services B.V.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ */
+package org.opensmartgridplatform.adapter.protocol.dlms.application.services;
+
+import java.util.Timer;
+import java.util.TimerTask;
+import java.util.concurrent.Semaphore;
+
+import javax.annotation.PostConstruct;
+import javax.annotation.PreDestroy;
+
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import org.springframework.beans.factory.annotation.Value;
+import org.springframework.stereotype.Component;
+
+@Component
+public class ThrottlingService {
+
+    private static final Logger LOGGER = LoggerFactory.getLogger(ThrottlingService.class);
+
+    public void setMaxOpenConnections(int maxOpenConnections) {
+        this.maxOpenConnections = maxOpenConnections;
+    }
+
+    public void setMaxNewConnectionRequests(int maxNewConnectionRequests) {
+        this.maxNewConnectionRequests = maxNewConnectionRequests;
+    }
+
+    public void setResetTime(int resetTime) {
+        this.resetTime = resetTime;
+    }
+
+    @Value("${throttlingService.maxOpenConnections}")
+    private int maxOpenConnections;
+    @Value("${throttlingService.maxNewConnectionRequests}")
+    private int maxNewConnectionRequests;
+    @Value("${throttlingService.resetTime}")
+    private int resetTime;
+
+    private Semaphore openConnectionsSemaphore;
+    private Semaphore newConnectionRequestsSemaphore;
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Move this variable to comply with Java Code Conventions. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](http://ci.opensmartgridplatform.org/sonarqube/coding_rules#rule_key=squid%3AS1213)